README.md

```diff
@@ -366,6 +366,7 @@ Hurray! You successfully made a contribution!
   - [Richard Bacon](https://github.com/RichardJonBacon)
   - [Rick Flores](https://github.com/RICK-FLORES)
   - [Rishabh Singhal](https://github.com/rish-singhal)
+  - [Rishabh Rathore](https://github.com/rishabhrathore055)
```

Comment:
Please add yourself one line above R is bevor s in the last name

Reply:
@schmelto Sure